test/spec/features/context-pad/ContextPadProviderSpec.js

```diff
@@ -489,7 +489,7 @@ describe('features - context-pad', function() {
 
           // given
           var rootShape = canvas.getRootElement(),
-              dataObject = elementFactory.createShape({ type: 'bpmn:DataObjectReference' }),
+              dataObject = elementFactory.createShape({ type: 'bpmn:DataStoreReference' }),
```

Comment:
This does not seem right: 

```
dataObject = elementFactory.createShape({ type: 'bpmn:DataStoreReference' })
```

Reply:
True, it was originally a dataObject but the test was failing now that dataObjectReferences can have a replace menu.
I forgot to rename the variable as well.